velox/exec/TableWriter.h

```diff
@@ -26,6 +26,29 @@ namespace facebook::velox::exec {
 class TableWriterTraits {
  public:
   /// Defines the column channels in table writer output.
+  /// Both the statistics and the row_count + fragments are transferred over the
```

Comment:
Can you rephrase the comment
```
(1) output layout
(2) row count + fragment and stats are not both set? Explain that we first yield the aggregation stats?
```

Reply:
Actually those are already in the comment in the later paragraph

Reply:
Can you first comment on the layout and then describe the special treatments on the output?